api/fixtures/example.go

```diff
@@ -298,6 +300,30 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 		default:
 			panic(fmt.Sprintf("service publishing type %s is not supported", o.Kubevirt.ServicePublishingStrategy))
 		}
+		if o.Kubevirt.ExtClusterKubeConfig.KubeConfigContents != nil && o.Kubevirt.ExtClusterKubeConfig.ClusterName != "" {
+			extKubeConfigSecret := &corev1.Secret{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "Secret",
+					APIVersion: corev1.SchemeGroupVersion.String(),
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: o.Namespace,
+					Name:      o.Kubevirt.ExtClusterKubeConfig.ClusterName + "-kubeconfig",
+				},
+				Data: map[string][]byte{
+					"kubeconfig": []byte(base64.StdEncoding.EncodeToString(o.Kubevirt.ExtClusterKubeConfig.KubeConfigContents)),
+				},
+			}
+			resources = append(resources, extKubeConfigSecret)
+			platformSpec.KubeVirt = &hyperv1.KubeVirtPlatformSpec{
+				ExtClusterName: o.Kubevirt.ExtClusterKubeConfig.ClusterName,
+				KubeConfigSecret: &hyperv1.KubeconfigSecretRef{
+					Name: extKubeConfigSecret.Name,
+					Key:  hostedcontrolplane.DefaultAdminKubeconfigKey,
```

Comment:
`Key:  hostedcontrolplane.DefaultAdminKubeconfigKey,` - i don't understand that part.

Reply:
it is the key of the Secret object that contains the kubeconfig. the key is `kubeconfig` and it is used as a const at `hostedcontrolplane.DefaultAdminKubeconfigKey` and I reused it.
But now i removed it, for clarity.